test/test_unit_init_controller.py

```diff
@@ -77,7 +78,7 @@ def test_init_not_trusting_cert_raises_error(self, mock_input):
         self.init_logic.connect = MagicMock(return_value = sock)
         self.init_logic.get_certificate = MagicMock(return_value = ["12:AB", "somecertchain"])
 
-        with self.assertRaises(RuntimeError):
+        with self.assertRaises(CertificateNotTrust):
```

Comment:
rename here as well to Error\Exception suffix

Reply:
`CertificateNotTrust` -> `CertificateNotTrustedException` is cleaner IMO

Reply:
done